keywords/remoteexecutor.py

```diff
@@ -43,12 +43,11 @@ def execute(self, command):
 
         log_info("Connecting to {}".format(self.host))
         log_info("Running '{}' on host {}".format(command, self.host))
-
         if self.sg_platform == "windows":
             self.client.connect(self.host, username=self.username, password=self.password, banner_timeout=REMOTE_EXECUTOR_TIMEOUT)
             command = "cmd /c " + command
             stdin, stdout, stderr = self.client.exec_command(command, timeout=60)
-        elif "c" in self.sg_platform:
+        elif "c-" in self.sg_platform:
```

Comment:
Is it possible to make change as c-platform as I requested in review? Also, make a note that this variable used says sg-platform not cbl-platforms

Reply:
I tried changing it , but it didn't look good and after rethinking it.

`cbl_ platform = self.sg_platform 
 if self.sg_platform == "windows":
 
elif "c-" in cbl_ platform:

elif self.sg_platform == "mac"
`
We can add a comment above the condition saying for CBL.
Or change the SG_platform variable in the class to just platform .

Reply:
For now, I can accept this. We need to figure out a better way , As 'c-' will be so common for future OS and we might accidentally add something like 'mac-arm' or it could be anything with 'c-'. To make it unique to c platforms ,you can always add 'cplatform-ios' or something similar, so that it won't conflict with c-platform logic. It is additional work later.